docs/reference/configuration.md

```diff
@@ -170,6 +170,12 @@ The hostname that the dev server is running on. Snowpack uses this information t
 
 The port the dev server runs on.
 
+### devOptions.urlpath
```

Comment:
nit: can we call this `openUrl`? `urlpath` is just vague enough that I'd be worried about users confusing it with `baseUrl`

Reply:
agree on finding the right name! Updated.